ui/component/settingCommentsServer/view.jsx

```diff
@@ -0,0 +1,69 @@
+// @flow
+import React from 'react';
+import Comments from 'comments';
+import { FormField } from 'component/common/form';
+
+const DEBOUNCE_TEXT_INPUT_MS = 500;
+
+type Props = {
+  customServerEnabled: boolean,
+  customServerUrl: string,
+  setCustomServerEnabled: (boolean) => void,
+  setCustomServerUrl: (string) => void,
+};
+
+function SettingCommentsServer(props: Props) {
+  const { customServerEnabled, customServerUrl, setCustomServerEnabled, setCustomServerUrl } = props;
+  const [customUrl, setCustomUrl] = React.useState(customServerUrl);
+
+  React.useEffect(() => {
+    const timer = setTimeout(() => {
+      setCustomServerUrl(customUrl);
+      Comments.url = customUrl;
+    }, DEBOUNCE_TEXT_INPUT_MS);
+
+    return () => clearTimeout(timer);
+  }, [customUrl, setCustomServerUrl]);
+
+  return (
+    <React.Fragment>
+      <fieldset-section>
+        <FormField
+          type="radio"
+          name="use_default_comments_server"
+          label={__('Use official Odysee comments server')}
```

Comment:
`__('Default comments server (%DEFAULT_COMMENT_SERVER_NAME%)', { DEFAULT_COMMENT_SERVER_NAME })`

with the default server name moved to a config

Reply:
alternatively, add an array config value for all known servers, and make this a select with custom as the last option